app/domain/authentication/authn_azure/application_identity.rb

```diff
@@ -0,0 +1,118 @@
+module Authentication
+  module AuthnAzure
+
+    Log = LogMessages::Authentication
```

Comment:
Is this on the `Authentication` module on purpose?

Reply:
At current, the log messages are the same ones used by k8s so I moved them up to general Authentication logs. For example, `RetrievedAnnotationValue`, `ValidatingAnnotationsWithPrefix`. When I get into the more specific authn-azure logs, I will reference them differently